resources/lib/ipwww_common.py

```diff
@@ -213,13 +214,14 @@ def OpenURL(url):
     try:
         for cookie in r.cookies:
             cookie_jar.set_cookie(cookie)
```

Comment:
I just noticed this line. Why didn't you replace cookie_jar with common_cookie_jar here?